src/Tokenizers/PHP.php

```diff
@@ -870,6 +879,35 @@ protected function tokenize($string)
                 continue;
             }//end if
 
+            /*
+                Enum keyword for PHP < 8.1
+            */
+
+            if ($tokenIsArray === true
+                && $token[0] === T_STRING
+                && strtolower($token[1]) === 'enum'
+            ) {
+                // Get the next non-empty token.
+                for ($i = ($stackPtr + 1); $i < $numTokens; $i++) {
+                    if (is_array($tokens[$i]) === false
+                        || isset(Util\Tokens::$emptyTokens[$tokens[$i][0]]) === false
+                    ) {
+                        break;
+                    }
+                }
+
+                if (isset($tokens[$i]) === true
+                    && is_array($tokens[$i]) === true
+                    && $tokens[$i][0] === T_STRING
+                ) {
+                    $tokens[$stackPtr][0] = T_ENUM;
+                    $token[0] = T_ENUM;
+                    if (PHP_CODESNIFFER_VERBOSITY > 1) {
+                        echo "\t\t* token $stackPtr changed from T_STRING to T_ENUM".PHP_EOL;
+                    }
```

Comment:
Just checking: why not assign the token to the `$finalTokens` array and `continue` onto the next token ?

Reply:
Thanks, modified.